### PR TITLE
Laravel 6 bump

### PR DESCRIPTION
**Pull request by SiebeVE**
This closes issue #21

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,14 +15,14 @@
   "require": {
     "php": ">=7.1",
     "vlucas/phpdotenv": "3.*",
-    "illuminate/console": "5.*",
-    "illuminate/support": "5.*"
+    "illuminate/console": "5.*|^6",
```
Comment (patrickbrouwers) [inline composer.json]: I think Laravel suggests to use `^6.0`

File: composer.json [hunk 2]
```diff
@@ -15,14 +15,14 @@
...
+    "illuminate/support": "5.*|^6"
   },
   "require-dev": {
-    "phpunit/phpunit": "^7.5",
+    "phpunit/phpunit": "^7.5|^8.3",
     "mikey179/vfsstream": "^1.6",
     "mockery/mockery": "^1.0",
-    "orchestra/testbench": "~3.8"
+    "orchestra/testbench": "~3.8|^4"
```
Comment (patrickbrouwers) [inline composer.json]: Orchestra testbench suggests using `^4.0`

File: composer.json (hunk 1 above)
Comment (SiebeVE) [reply composer.json]: Okay, edited!

File: composer.json (hunk 2 above)
Comment (SiebeVE) [reply composer.json]: Okay, edited!

**Comment by JulienTant** [comment]
Thank you!

**Pull request merged**